app/grants/templates/grants/cart/extension/kusama.html

```diff
@@ -115,7 +115,7 @@
                     <span class="font-weight-semibold">Balance:</span>
                     <span class="clr-match-box py-1 font-smaller-7 px-2">
                       ([[ address_info.balance ]] [[ address_info.token_symbol ]])
-                      <template v-if="!address_info.sufficent_balance">Insufficent Balance</template>
+                      <template v-if="!address_info.sufficent_balance">insufficient Balance</template>
```

Comment:
~insufficient Balance~ Insufficient Balance (title case)

here and 8 other occurrences